sdk/js/src/util/marshaler.js

```diff
@@ -82,12 +90,51 @@ class Marshaler {
     return this.payloadSingleResponse(result, transform)
   }
 
+  static unwrapDevices (result, transform) {
+    return this.payloadListResponse(result, transform)
```

Comment:
```suggestion
    return this.payloadListResponse('devices', result, transform)
```
you are passing the `transform` function instead of the `result` object to `payloadListResponse`